packages/legend-application-studio/src/stores/LegendStudioApplicationNavigationContext.ts

```diff
@@ -40,10 +40,17 @@ export enum LEGEND_STUDIO_APPLICATION_NAVIGATION_CONTEXT_KEY {
   FUNCTION_EDITOR = 'studio.editor.function-editor',
 
   MAPPING_EDITOR = 'studio.editor.mapping-editor',
+  CLASS_MAPPING_EDITOR = 'studio.editor.mapping-editor.class-mapping-editor',
+  OPERATION_CLASS_MAPPING_EDITOR = 'studio.editor.mapping-editor.operation-class-mapping-editor',
+  ENUMERATION_MAPPING_EDITOR = 'studio.editor.mapping-editor.enumeration-mapping-editor',
+  MAPPING_EXECUTION_EDITOR = 'studio.editor.mapping-editor.execution-editor',
+  MAPPING_TEST_EDITOR = 'studio.editor.mapping-editor.test-editor',
+
   RUNTIME_EDITOR = 'studio.editor.runtime-editor',
   CONNECTION_EDITOR = 'studio.editor.connection-editor',
```

Comment:
can we add connection editor database builder :)

Reply:
of course!